x-pack/plugin/esql/qa/testFixtures/src/main/resources/lookup.csv-spec

```diff
@@ -1,5 +1,117 @@
+keywordByInt
+required_capability: lookup
+FROM employees
+| SORT emp_no
+| LIMIT 4
+| RENAME languages AS int
+| LOOKUP int_number_names ON int
+| RENAME int AS languages, name AS lang_name
+| KEEP emp_no, languages, lang_name
+;
+
+emp_no:integer | languages:integer | lang_name:keyword
+         10001 |                 2 | two
+         10002 |                 5 | five
+         10003 |                 4 | four
+         10004 |                 5 | five
+;
+
+keywordByMvInt
+required_capability: lookup
+ROW int=[1, 2, 3]
+| LOOKUP int_number_names ON int
+;
+
+int:integer | name:keyword
+  [1, 2, 3] | [one, two, three]
+;
+
+intByKeyword
+required_capability: lookup
+ROW name="two"
+| LOOKUP int_number_names ON name
+;
+
+name:keyword | int:integer 
+         two | 2
+;
+
+
+keywordByLong
+required_capability: lookup
+FROM employees
+| SORT emp_no
+| LIMIT 4
+| RENAME languages.long AS long
+| LOOKUP long_number_names ON long
+| RENAME long AS languages, name AS lang_name
+| KEEP emp_no, languages, lang_name
+;
+
+emp_no:integer | languages:long | lang_name:keyword
+         10001 |              2 | two
+         10002 |              5 | five
+         10003 |              4 | four
+         10004 |              5 | five
+;
+
+longByKeyword
+required_capability: lookup
+ROW name="two"
+| LOOKUP long_number_names ON name
+;
 
+name:keyword | long:long
+         two |         2
+;
 
+keywordByFloat
+required_capability: lookup
+FROM employees
+| SORT emp_no
+| LIMIT 4
+| RENAME height AS double
+| LOOKUP double_number_names ON double
+| RENAME double AS height, name AS height_name
+| KEEP emp_no, height, height_name
+;
+
+emp_no:integer | height:double | height_name:keyword
+         10001 |          2.03 | two point zero three
+         10002 |          2.08 | two point zero eight
+         10003 |          1.83 | null
+         10004 |          1.78 | null
+;
+
+floatByKeyword
+required_capability: lookup
+ROW name="two point zero eight"
+| LOOKUP double_number_names ON name
+;
+
+    name:keyword     | double:double
+two point zero eight |          2.08
+;
+
+intIntByKeywordKeyword
+required_capability: lookup
+ROW aa="foo", ab="zoo"
```

Comment:
Please add a test where aa and ab (as lookup keys) have the same value - `row aa = "foo", ab="foo" | lookup big on aa, ab".
Followed by variations:
 - lookup on field with empty value
 - lookup on function over field (should give validation error) - can be a follow-up.

Reply:
"lookup on function over field:" - you mean like `LOOKUP foo ON CONCAT(a, b)`?